frontendcomponent/androidcomponent/src/main/res/layout/section.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:orientation="vertical"
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content">
+    <TextView
+        android:id="@+id/section_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@color/gray1"
+        android:textSize="24sp"
+        android:text="@string/session_label"
+        android:layout_marginTop="22dp"
+        android:layout_marginStart="16dp"
+        android:layout_marginBottom="27dp"
```

Comment:
it looks like maybe this should be `28dp` 🤔 
From [Material.io](https://material.io/design/layout/spacing-methods.html#spacing)

> Padding refers to the space between UI elements. It’s an alternative spacing method to keylines, measured in increments of 8dp or 4dp.

So it looks better to be `28dp` I think!

<img width="387" alt="screen shot 2019-01-22 at 22 37 08" src="https://user-images.githubusercontent.com/1574446/51539359-23bd1500-1e97-11e9-88ac-b52db0e2bf80.png">

Reply:
sorry 🙇inspect then this.

<img width="360" alt="2019-01-22 22 50 34" src="https://user-images.githubusercontent.com/11706821/51539841-561b4200-1e98-11e9-97fd-c5ab24da1907.png">


So I was doing it at 27 dp. but, I will change 28dp. Thank you 😄